R/utils.R

```diff
@@ -200,3 +200,54 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print Information About A Forecast Object
+#' @description This function prints information about a forecast object,
+#' including "Forecast type", "Score columns",
+#' "Forecast unit".
+#'
+#' @param x An object of class 'forecast_*' object as produced by
+#' `as_forecast()`
+#' @param ... additional arguments for [print()]
+#' 
```

Comment:
```suggestion
```

Reply:
There is a setting in Rstudio where you can automatically remove trailing whitespaces on every save. I'm using that and it's really nice.

Reply:
I am using VS code, and now I turn on the function of automatically removing all trailing whitespace. Thanks for the suggestion.